### PR TITLE
CNV-24126: Rearrange vm tabs

### PR DESCRIPTION
**Pull request by metalice**
## 📝 Description

We are rearranging tabs according to the new design.
https://docs.google.com/document/d/12VohNCqxqJP4TYzAO2WJvAYT3OPkfRGjez4rrRLuyZE/edit#heading=h.vcrz1bis83iy

## 🎥 Demo

image

### CODE REVIEW

File: src/utils/components/PendingChanges/utils/constants.ts [hunk 1]
```diff
@@ -39,6 +40,8 @@ export enum VirtualMachineDetailsTabLabel {
   Snapshots = 'Snapshots',
   // t('Scripts')
   Scripts = 'Scripts',
-  // t('Diagnostic)
+  // t('Diagnostic')
   Diagnostic = 'Diagnostic',
+  // t('Configurations')
+  Configurations = 'Configurations',
```
Comment (pcbailey) [inline src/utils/components/PendingChanges/utils/constants.ts:46]: I believe this should be 'Configuration'.

**Review by pcbailey — COMMENTED** [review]
Could you link to the design document? I think part of the review should be checking the final result against the design to help each catch anything we may have missed.

**Comment by metalice** [comment]
> Could you link to the design document? I think part of the review should be checking the final result against the design to help each catch anything we may have missed.

Added in the description the link

File: src/utils/components/PendingChanges/utils/constants.ts (hunk 1 above)
Comment (metalice) [reply src/utils/components/PendingChanges/utils/constants.ts:46]: Well in the design its Configurations but you know English, lol. will change

**Comment by upalatucci** [comment]
/retest

for me its lgtm

**Pull request merged**